packages/input-iban/translations/hu.js

```diff
@@ -16,5 +16,20 @@ export default {
       'RO {Román}\n' +
       'other {{params}}\n' +
       '} {fieldName} értéket.',
+    IsNotCountryIBAN:
+      '{params, select,\n' +
+      'AT {Osztrák}\n' +
+      'BE {Belga}\n' +
+      'CZ {Cseh}\n' +
+      'DE {Német}\n' +
+      'ES {Spanyol}\n' +
+      'FR {Francia}\n' +
+      'HU {Magyar}\n' +
+      'IT {Olasz}\n' +
+      'NL {Holland}\n' +
+      'PL {Lengyel}\n' +
+      'RO {Román}\n' +
+      'other {{params}}\n' +
+      '} {fieldName} nem engedélyezett',
```

Comment:
Its a sentence, so should end with a `.`. (also for all following languages)

Reply:
Done